packages/orchestrator-ui-components/src/components/WfoForms/formFields/AcceptField.tsx

```diff
@@ -0,0 +1,239 @@
+/*
+ * Copyright 2019-2023 SURF.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import { EuiCheckbox, EuiFlexItem, EuiText } from '@elastic/eui';
+import React, { useReducer } from 'react';
+import { connectField, filterDOMProps } from 'uniforms';
+
+import { FieldProps } from './types';
+import { useTranslations } from 'next-intl';
+import { useOrchestratorTheme } from '../../../hooks';
+import { getStyles } from './AcceptFieldStyling';
+
+type AcceptItemType =
+    | 'info'
+    | 'label'
+    | 'warning'
+    | 'url'
+    | 'checkbox'
+    | '>checkbox'
+    | 'checkbox?'
+    | '>checkbox?'
+    | 'skip'
+    | 'margin'
+    | 'value';
+type AcceptItem = [string, AcceptItemType, Record<string, string>?];
+type AcceptValue = 'SKIPPED' | 'ACCEPTED' | 'INCOMPLETE';
+
+export type AcceptFieldProps = FieldProps<AcceptValue, { data?: AcceptItem[] }>;
+
+declare module 'uniforms' {
+    interface FilterDOMProps {
+        data: never;
+    }
+}
+
+filterDOMProps.register('data');
+
+interface AcceptState {
+    checks: { [index: number]: boolean };
+    skip: boolean;
+    allChecked: boolean;
+}
+
+interface Action {
+    field: number;
+    type: string;
+    value: boolean;
+}
+
+function Accept({
+    disabled,
+    className = '',
+    name,
+    onChange,
+    error,
+    showInlineError,
+    errorMessage,
+    data,
+    ...props
+}: AcceptFieldProps) {
+    const t = useTranslations('pydanticForms.fields');
+    const { theme } = useOrchestratorTheme();
```

Comment:
So you could use the useWithOrchestrator theme here to safe aline of code

Reply:
✅